tables/da-dk-octobraille.dis

```diff
@@ -22,22 +22,25 @@
 #
 # This file contains display definitions for Danish 8 dot Braille.
 # It should be included in high level tables to ensure correct mapping of characters to dot patterns.
+# See also: Den Danske Punktskrift 2022
+#           Det Danske Punktskriftsnævn
+#           https://www.blind.dk/punktskrift-2022
```

Comment:
@BueVest I'm stumbling a bit over the fact that a revision of the braille translation rules also results in a change of this display table. What exactly is "Octobraille"? Is it defined in the DDP? You referenced https://www.blind.dk/punktskrift-2022 but I'm not sure which of the links on that page actually define it.